ClimsoftVer4/ClimsoftVer4/ucrValueFlagPeriod.vb

```diff
@@ -264,6 +265,12 @@ Public Class ucrValueFlagPeriod
         Return bValuesCorrect
     End Function
 
+    Private Sub SetTextBoxSize()
+        ucrValue.txtBox.Size = New Size(51, 20)
+        ucrFlag.txtBox.Size = New Size(27, 20)
+        ucrPeriod.txtBox.Size = New Size(33, 20)
```

Comment:
We can have 

ucrValue.txtBox.Size = New Size(51, 20)

be like

ucrValue.SetElementValueSize(New Size(51, 20))

by creating the above subroutine in the control. This will expose this functionality to other classes and promote encapsulation principle of OOP.